pilot/views.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+from student.models import Student, PilotOffering
+from timetable.models import CourseIntegration, Course, Section, Semester
+from django.shortcuts import get_object_or_404, render, redirect
+from .forms import StudentForm
+from django.db import transaction
+import semesterly.views
+
+
+def index(request, id):
+	if request.method == 'POST':
+		return redirect('pilot:info', id=id)
+	else:
+		student,created = Student.objects.get_or_create(id=id)
+		context = {
+			'student': student
+		}
+		return render(request, 'pilot/welcome.html/', context=context)
+
+
+def info(request, id):
+	if request.method == 'POST':
+		form = StudentForm(request.POST)
+		student_object, created = Student.objects.get_or_create(id=id)
+		print form.errors
```

Comment:
did you leave the print statement intentionally or was it for your local debugging purposes?

Reply:
It was for debugging! Deleting print statements!